CHANGELOG.rst

```diff
@@ -2,6 +2,289 @@
+* Fixed examples in the documentation for logic with date and duration calculations.
+* [:backend:`4745`] Fixed missing registration variable to the Objects API with all
+  the attachment URLs.
+* [:backend:`4823`] Fixed uploaded files with leading or trailing whitespaces in the
+  filename.
+* [:backend:`4810`] Fixed uppercase component variable values turing lowercase.
+* [:backend:`4772`] Fixed select components with integer values being treated as numbers
+  instead of strings.
+* [:backend:`4727`] Fixed crash when a user defined variable was changed to an array
+  datatype.
+* Fixed type error in the preset nested validate schema for components.
+* [:backend:`4802`] Fixed some dropdowns taking up more horizontal space than intended.
+* [:backend:`4763`] Fixed temporary file uploads not being delete-able in the admin interface.
+* [:backend:`4726`] Fixed the styling for form delete buttons.
+* [:backend:`4744`] Fixed a performance regression in the logic check calls and general
+  submission processing.
+* [:backend:`4774`] Fixed ``textfield`` data not being converted to a string when numeric
+  data is received from a prefill plugin.
+* Fixed docs concerning invalid SSL certs and broken links.
+* [:backend:`4765`] Fixed bug in components migration converter when multiple is True.
+* [:backend:`4546`] Fixed the soft-required validation errors being shown in the summary PDF.
+* Fixed validation error when saving a new form definition via the admin.
+* [:backend:`4659`] Fixed ``null`` default values for text-based fields.
+* [:backend:`4528`] Fixed vague error/log out situation when logging in with OIDC.
+* [:backend:`3629`] Fixed submission bulk export crashing when the form has repeating
+  groups.
+* [:backend:`3705`] Updated timestamps in str representations.
+* [:backend:`4713`] Fixed pre-request hook not running for all "Haal Centraal BRP
+  Personen bevragen" operations (fixes Token Exchange extension).
+* [:backend:`4600`] Fixed not all the content on the page getting translated after changing
+  the form language.
+* [:backend:`4733`] Fixed a segmentation fault that could occur in dev environments.
+* [:backend:`4628`] Fixed a crash when copying a form with a "block next step" logic.
+* [:backend:`4711`] Fixed broken submission form row styling.
+* [:backend:`4695`] Fixed a performance issue during legacy Objects API registration
+  plugin validation.
+* [:backend:`4652`] Fixed misaligned validation errors in the form designer UI.
+* [:backend:`4658`] Fixed certain variants of ZIP files not passing validation on Windows.
+* [:backend:`4656`] Fixed a crash during validation when you have file upload components
+  inside repeating groups.
+
+**Project maintenance**
+
+* Fixed stories concerning bootstrap.
+* Removed automatic patching for `cosign_information` template tag.
+  (see **Installation** > **Upgrade details to Open Forms 3.0.0**) for all the necessary details.
+* Updated browserslist package.
```

Comment:
Leave this one out, it just affects which CSS prefixed and modern-to-old-JS conversions are required.